### PR TITLE
Add a skip parameter to all readers (defaults to zero)

### PR DESCRIPTION
**Pull request by rantav**
Followup on https://github.com/huggingface/datatrove/issues/164

### CODE REVIEW

File: src/datatrove/pipeline/readers/parquet.py [hunk 1]
```diff
@@ -33,6 +33,7 @@ def __init__(
         self,
```
Comment (hynky1999) [inline src/datatrove/pipeline/readers/parquet.py:33]: Could you also add the limit section in docstring above ?

**Review by hynky1999 — COMMENTED** [review]
LGTM!

I just have one very small nitpick regarding docstring.
Thank you for adding tests

File: src/datatrove/pipeline/readers/parquet.py (hunk 1 above)
Comment (rantav) [reply src/datatrove/pipeline/readers/parquet.py:34]: Yes, done, thank you for the quick review!

**Review by hynky1999 — APPROVED** [review]

**Comment by guipenedo** [comment]
ty :)

**Pull request merged**